ReactNative/js/bridge-manager.js

```diff
@@ -17,43 +18,44 @@ export default class BridgeManager {
   }
 
   async onAction({ module, action, args, id }) {
-    for(const listener of this.actionListeners) {
+    [...this.actionListeners].some(listener => {
       try {
         const handled = listener({ module, action, args, id });
         if (handled) {
-          return;
+          return false;
```

Comment:
Shouldn't this be `true`? Meaning that the first time a listener handles an action you stop.